apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/issues/api.js

```diff
@@ -37,9 +37,9 @@ export default {
             data: {startEpoch: startTimeStamp, endTimeStamp}
         })
     },
-    fetchTestCoverageData (startTimeStamp, endTimeStamp) {
```

Comment:
see if this API is being called anywhere else in the UI. If yes handle accordingly.